ui/app/components/clients/page/sync.hbs

```diff
@@ -2,58 +2,68 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
+{{#if @secretsSyncActivated}}
+  {{#if this.isDateRange}}
+    <Clients::ChartContainer
+      @title={{this.title}}
+      @description={{this.description}}
+      @timestamp={{@activity.responseTimestamp}}
+      @hasChartData={{this.byMonthActivityData}}
+      class="no-legend"
+    >
+      <:subTitle>
+        <StatText
+          @label="Total sync clients"
+          @subText="The total number of secrets synced from Vault to other destinations during this date range."
+          @value={{this.totalUsageCounts.secret_syncs}}
+          @size="l"
+          data-test-total-sync-clients
+        />
+      </:subTitle>
 
-{{#if this.isDateRange}}
-  <Clients::ChartContainer
-    @title={{this.title}}
-    @description={{this.description}}
-    @timestamp={{@activity.responseTimestamp}}
-    @hasChartData={{this.byMonthActivityData}}
-    class="no-legend"
-  >
-    <:subTitle>
-      <StatText
-        @label="Total sync clients"
-        @subText="The total number of secrets synced from Vault to other destinations during this date range."
-        @value={{this.totalUsageCounts.secret_syncs}}
-        @size="l"
-        data-test-total-sync-clients
-      />
-    </:subTitle>
+      <:stats>
+        <StatText
+          @label="Average sync clients per month"
+          @value={{this.average this.byMonthActivityData "secret_syncs"}}
+          @size="m"
+          class="data-details-top has-top-padding-l"
+          data-test-average-sync-clients
+        />
+      </:stats>
 
-    <:stats>
-      <StatText
-        @label="Average sync clients per month"
-        @value={{this.average this.byMonthActivityData "secret_syncs"}}
-        @size="m"
-        class="data-details-top has-top-padding-l"
-        data-test-average-sync-clients
-      />
-    </:stats>
+      <:chart>
+        <Clients::Charts::VerticalBarBasic
+          @chartTitle={{this.title}}
+          @data={{this.byMonthActivityData}}
+          @dataKey="secret_syncs"
+          @chartHeight={{200}}
+        />
+      </:chart>
 
-    <:chart>
-      <Clients::Charts::VerticalBarBasic
-        @chartTitle={{this.title}}
-        @data={{this.byMonthActivityData}}
-        @dataKey="secret_syncs"
-        @chartHeight={{200}}
-      />
-    </:chart>
-
-    <:emptyState>
-      <EmptyState
-        @title="No monthly secrets sync clients"
-        @subTitle="There is no secrets sync data available for this date range."
-        @bottomBorder={{true}}
-      />
-    </:emptyState>
-  </Clients::ChartContainer>
-{{else}}
-  <div class="chart-wrapper" data-test-usage-stats>
-    <div class="chart-header has-bottom-margin-m">
-      <h2 class="chart-title">{{this.title}}</h2>
-      <p class="chart-description has-bottom-padding-m">{{this.description}}</p>
+      <:emptyState>
+        <EmptyState
+          @title="No monthly secrets sync clients"
+          @subTitle="There is no secrets sync data available for this date range."
```

Comment:
@hellobontempo again, apologies defaulting to you as you have spent a lot of time in this area. Should this be `Secrets Sync` instead of lower case?

Reply:
Good question, but again this isn't referring to the feature so it isn't a proper noun and should remain lower cased like `entity clients` and `non-entity clients`